cdc/sink/codec/canal.go

```diff
@@ -126,6 +129,7 @@ func (b *canalEntryBuilder) buildHeader(commitTs uint64, schema string, table st
 // build the Column in the canal RowData
 func (b *canalEntryBuilder) buildColumn(c *model.Column, colName string, updated bool) (*canal.Column, error) {
 	sqlType := MysqlToJavaType(c.Type)
+	mysqlType := parser_types.TypeToStr(c.Type, c.Charset)
```

Comment:
You only need to determine whether Charset is 'binary' or 'utf8'.

Reply:
You means that we don't need mysqlType in canal column now?